scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/InProcessTaskExecutor.java

```diff
@@ -156,6 +160,14 @@ public TaskResultImpl execute(TaskContext taskContext, PrintStream output, Print
             if (nodesFile != null && !nodesFile.isEmpty()) {
                 FileUtils.deleteQuietly(new File(nodesFile));
             }
+            if (client != null) {
+                if (client.isConnected()) {
+                    try {
+                        client.disconnect();
+                    } catch (Exception ignored) {
```

Comment:
We cannot trace in loggers as this is executed inside the forked JVM. I could put a warning message eventually